apps/rush-lib/src/rush.schema.json

```diff
@@ -149,5 +164,53 @@
     }
   },
   "additionalProperties": false,
-  "required": [ "npmVersion", "rushMinimumVersion", "projects" ]
+  "required": [ "npmVersion", "rushMinimumVersion", "projects" ],
+  "definitions": {
+    "lockStepVersion": {
+      "type":"object",
+      "description": "Lockstep version policy",
+      "properties": {
+        "policyName": {
+          "description": "The name of the version policy",
+          "type": "string"
+        },
+        "baseType": {
+          "description": "The name of base version policy",
+          "type": "string",
+          "enum": [ "lockStepVersion" ]
+        },
+        "version": {
+          "description": "Current version for projects with lockStepVersion policy",
+          "type":"string"
+        },
+        "nextBump": {
+          "description": "Type of next version bump",
+          "enum": [ "prerelease", "release", "minor", "patch", "major" ]
+        }
```

Comment:
I am trying to avoid abstracting thing unnecessarily here.

---
In reply to: [125352546](https://github.com/Microsoft/web-build-tools/pull/259#discussion_r125352546) [](ancestors = 125352546)